### ISSUE TITLE
"EXAMPLE 9: definition of aboutUrl using column names" not "titles"

### ISSUE THREAD

**Issue by Robsteranium** [w3c/csvw#847]
On the Metadata Vocabulary, should the heading "EXAMPLE 9: definition of aboutUrl using column titles" instead read "... using column names"?

Sorry if this is a bit pedantic but since `csvw:title` is distinct from `csvw:name` I thought it worth clarifying: should the template use a reference to the column name rather than it's natural language title?

The example itself seems to correspond with this.

**Comment by gkellogg** [comment]
@Robsteranium Yes, you're correct. The text under **column names** describes that it is the names which are used as variables when resolving URI Template Properties (in addition to special variables like `_column`), not the titles.

The title for EXAMPLE 9 should be "definition of aboutUrl using column names".
